.github/actions/test_tests-cache.sh

```diff
@@ -1,22 +1,15 @@
 #!/bin/bash
 set -e -u -x -o pipefail
 
-TMP_CACHE_DIR=$(mktemp -d -t glpi-cache-test-XXXXXXXXXX)
+PHPUNIT_ADDITIONNAL_OPTIONS=""
+if [[ "${CODE_COVERAGE:-}" = true ]]; then
+  export COVERAGE_DIR="coverage-functional"
+  PHPUNIT_ADDITIONNAL_OPTIONS="--coverage-filter src --coverage-clover phpunit/$COVERAGE_DIR/clover.xml"
 
-for CONFIG in {"--use-default","--dsn=memcached://memcached","--dsn=redis://redis"}; do
-  php bin/console cache:configure \
-    --ansi --no-interaction \
-    $CONFIG
-  vendor/bin/atoum \
-    -p 'php -d memory_limit=512M' \
-    --debug \
-    --force-terminal \
-    --use-dot-report \
-    --bootstrap-file tests/bootstrap.php \
-    --fail-if-void-methods \
-    --fail-if-skipped-methods \
-    --no-code-coverage \
-    --max-children-number 1 \
-    -d tests/functional \
-    -t cache
-done
+else
+  PHPUNIT_ADDITIONNAL_OPTIONS="--no-coverage";
+fi
+
+vendor/bin/phpunit --group cache $PHPUNIT_ADDITIONNAL_OPTIONS $@
```

Comment:
The goal is to test all cache adapters.

```suggestion
for CONFIG in {"--use-default","--dsn=memcached://memcached","--dsn=redis://redis"}; do
  php bin/console cache:configure \
    --ansi --no-interaction \
    $CONFIG
  vendor/bin/phpunit --group cache $PHPUNIT_ADDITIONNAL_OPTIONS $@
done
```